libs/libGamedata/gameData/CampaignDescription.cpp

```diff
@@ -12,6 +13,7 @@
 CampaignDescription::CampaignDescription(const kaguya::LuaRef& table)
 {
     CheckedLuaTable luaData(table);
+    luaData.getOrThrow(uid, "uid");
```

Comment:
For this you need to  Increase `CampaignDataLoader::GetVersion()`

I'd try to make this backwards compatible: and make the uid optional. Or check the version here (again) and make sure the uid is empty for older versions and non-empty otherwise.

Then you can revert the changes to existing tests